src/components/vive-controls.js

```diff
@@ -105,6 +106,7 @@ module.exports.Component = registerComponent('vive-controls', {
     el.removeEventListener('touchstart', this.onButtonTouchStart);
     el.removeEventListener('model-loaded', this.onModelLoaded);
     el.removeEventListener('axismove', this.onAxisMoved);
+    this.controlllerEventsActive = false;
```

Comment:
Removing and adding controller specific event listeners is needed because we don't want to be adding Oculus Touch triggerdown when Oculus Touch controllers not connected, which would result in double-emit if we're using a Vive controller.